codespell_lib/data/dictionary.txt

```diff
@@ -6402,6 +6402,7 @@ confety->confetti
 conffiguration->configuration
 confgiuration->configuration
 confgiured->configured
+confguration->configuration
```

Comment:
Do you want to add -e and -ed variants of this too please?

Reply:
Sure.

Reply:
@peternewman I took the liberty to also add `confgiure->configure`, for completeness.

Reply:
I think that's what I meant by -e...

Reply:
I was referring to the existing `confgiur-` stem (different from this PR's original `confgur-` one), which already had the `-ure` and `-ed` endings, but not `-e`.

Reply:
Yeah, I think I realised that eventually...